### PR TITLE
Submit solution

### PR DESCRIPTION
**Pull request by lauraweston**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -88,3 +88,68 @@ Finally, don’t overcomplicate things. This task isn’t as hard as it may seem
...
 
 * Finally, please submit a pull request before Monday at 9am with your solution or partial solution.  However much or little amount of code you wrote please please please submit a pull request before Monday at 9am.
+
+Solution
+--------
+Using Rspec and TDD principles, I modelled the solution with classes for Airport, Plane and Weather. All instance variables remain private since it is unnecessary for the other classes to have access to them. I approached each user story one at a time, testing edge cases where necessary (e.g. planes cannot land/take off if already landed/airborne; an airport should release a specific plane at take off; an airport can only instruct a plane to take off if the plane is at the airport).
+
+**Assumptions:** Planes are airborne when initialized.
+
+**Generating random weather:** ```srand(1)``` is used in weather_spec.rb to seed a sequence of repeatable pseudo-random numbers between 1 and 5 that could be reused with each run of the rspec tests. Number 5 was used to denote stormy weather; all other numbers denote sunny weather. Random weather is generated by ```#current_weather```, which is called before each plane lands or takes off.
```
Comment (maryrosecook) [inline README.md]: Nice!

File: README.md [hunk 2]
```diff
@@ -88,3 +88,68 @@ Finally, don’t overcomplicate things. This task isn’t as hard as it may seem
 * **Submit a pull request early.**  There are various checks that happen automatically when you send a pull request.  **Fix these issues if you can**.  Green is good.
 
 * Finally, please submit a pull request before Monday at 9am with your solution or partial solution.  However much or little amount of code you wrote please please please submit a pull request before Monday at 9am.
+
+Solution
+--------
+Using Rspec and TDD principles, I modelled the solution with classes for Airport, Plane and Weather. All instance variables remain private since it is unnecessary for the other classes to have access to them. I approached each user story one at a time, testing edge cases where necessary (e.g. planes cannot land/take off if already landed/airborne; an airport should release a specific plane at take off; an airport can only instruct a plane to take off if the plane is at the airport).
```
Comment (maryrosecook) [inline README.md]: Great!

File: lib/airport.rb [hunk 3]
```diff
@@ -0,0 +1,39 @@
...
+    @planes.delete(plane)
+  end
+
+  def has_plane?(plane)
+    @planes.include?(plane)
+  end
+
+  private
+
+  def stormy?
```
Comment (lauraweston) [inline lib/airport.rb]: I've implemented these two changes now. Thanks for the feedback, Mary, really helpful!

Comment (maryrosecook) [inline lib/airport.rb]: My pleasure! Thanks for letting me know!

**Pull request closed without merging**